src/radical/entk/pipeline.py

```diff
@@ -406,9 +406,33 @@ def _validate(self):
 
             raise EnTKMissingError(obj=self._uid, missing_attribute='stages')
 
+        annotated_outputs = {}
         for stage in self._stages:
             stage._validate()  # pylint: disable=W0212
 
+            # validate provided annotations
+            for task in stage.tasks:
+
+                if not task.annotations:
+                    continue
+
+                annotated_outputs[task.uid] = task.annotations.outputs
+                for t_input in task.annotations.inputs:
+                    delimiter_count = t_input.count(':')
+                    if not delimiter_count:
+                        continue
+                    elif delimiter_count == 1 and '://' in t_input:
+                        # extend file representation format (future feature)
```

Comment:
Should we mark the comment with `TODO`, i.e., `# TODO: extend file representation format`? In that way, we would have a 'standardized' way to annotate future TODOs, and many IDEs can pick that up automatically.

Reply:
done